### PR TITLE
Add step to set up GOOGLE_APPLICATION_CREDENTIALS

### PR DESCRIPTION
**Pull request by gargnupur**

### CODE REVIEW

File: content/blog/2018/export-logs-through-stackdriver.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Common setup for all sinks:
 
 1. Enable [StackDriver Monitoring API](https://cloud.google.com/monitoring/api/enable-api) for the project.
 1. Make sure `principalEmail` that would be setting up the sink has write access to the project and Logging Admin role permissions.
+1. Make sure `GOOGLE_APPLICATION_CREDENTIALS` environment variable is set. Please follow instructions [here](https://cloud.google.com/docs/authentication/getting-started) to set it up.
```
Comment (geeknoid) [inline content/blog/2018/export-logs-through-stackdriver.md]: Make sure the

File: content/blog/2018/export-logs-through-stackdriver.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ Common setup for all sinks:
 
 1. Enable [StackDriver Monitoring API](https://cloud.google.com/monitoring/api/enable-api) for the project.
```
Comment (geeknoid) [inline content/blog/2018/export-logs-through-stackdriver.md]: StackDriver -> Stackdriver

Please check the rest of the doc for the correct capitalization.

**Comment by gargnupur** [comment]
@geeknoid : Am holding off on this review for now.. We have a meeting in the afternoon today to discuss if asking users to set GOOGLE_APPLICATION_CREDENTIALS is even the right approach or not...

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 45195ea7f96d719efa75454308748c4b4211f872

https://app.netlify.com/sites/preliminary-istio/deploys/5b296807c965921666b326f3

**Comment by geeknoid** [comment]
What's the status of this PR?

Thanks.

**Comment by gargnupur** [comment]
@geeknoid : I think it is.. 
@douglas-reid: Doug, can you please take a look too?

**Comment by geeknoid** [comment]
Hi, I did some refactoring and the file you edited has moved in the repo. Please rebase and push again. Sorry for the inconvenience.

**Comment by geeknoid** [comment]
@douglas-reid You happy with this PR?

**Comment by gargnupur** [comment]
@geeknoid : Np :) Finally, I re-based the PR without any issues!

**Comment by geeknoid** [comment]
This has broken links, remove the .md in the newly-added links.

**Pull request merged**